app/Http/Resources/V1/Admin/ProjectResource.php

```diff
@@ -29,8 +29,10 @@ public function toArray($request)
             'clone_ctr' => $this->clone_ctr,
             'users' => ProjectUserResource::collection($this->whenLoaded('users')),
             'playlists' => ProjectPlaylistResource::collection($this->whenLoaded('playlists')),
-            'is_public' => $this->is_public,
-            'elasticsearch' => $this->elasticsearch,
```

Comment:
Do we remove elasticsearch also?

Reply:
Yes. both fields were serving same purpose. So now we'll be managing it with "indexing" field. Because if user request for indexing is approved by admin then only it will be shown in elasticsearch so no need of separate field.